packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1760,6 +1776,7 @@ mixin _CopyPointerScrollEvent on PointerEvent {
     double? tilt,
     bool? synthesized,
     int? embedderId,
+    RespondPointerEventFn? respond,
```

Comment:
This should also follow the naming rules: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables